src/content/developers/tutorials/run-light-node-geth/index.md

```diff
@@ -162,6 +162,12 @@ eth.getBalance('0x85d918c2B7F172d033D190152AEc58709Fb6D048')
 
 ## Stopping and restarting your node {#stopping-and-restarting-your-node}
 
-You can stop your node at any time. If you want to restart the node, Geth will take a few seconds or minutes to re-synchronize (downloading the blocks and/or block headers from where it left off when the node last stopped running). If any of the above instructions aren’t working, the first thing you should do is try restarting your node.
+You can stop your node at any time by pressing `CTRL+C`. If you want to restart the node, Geth will take a few seconds or minutes to re-synchronize (downloading the blocks and/or block headers from where it left off when the node last stopped running). If any of the above instructions aren’t working, the first thing you should do is try restarting your node.
```

Comment:
Is this universal across operating systems? Or should we say something like "you can stop your node in the same way that you restart your server in your terminal" or something similar.

Reply:
`Ctrl+C` should work on Linux, macOS and afaik, Windows.
Looked this up to confirm and Ctrl+C should send a SIGINT (interrupt) command, but apparently it _can_ be reprogrammed, leaving Ctrl+Break as the backup.
https://docs.microsoft.com/en-us/windows/console/ctrl-c-and-ctrl-break-signals

I can update with this note, but I do agree with the original issue that there is currently no comment about how to stop a terminal process... Ctrl+C/Ctrl+Break should be plenty.